README.md

```diff
@@ -34,24 +34,29 @@ libraries and such might not want.
 
 ### Table of Contents <!-- omit in toc -->
 
-- [About](#about)
-- [Features](#features)
-- [Installing](#installing)
-- [Example](#example)
-- [Using](#using)
-  - [... in libraries](#-in-libraries)
-  - [... in application code](#-in-application-code)
-  - [... in `main()`](#-in-main)
-  - [... diagnostic code URLs](#-diagnostic-code-urls)
-  - [... snippets](#-snippets)
-  - [... multiple related errors](#-multiple-related-errors)
-  - [... delayed source code](#-delayed-source-code)
-  - [... handler options](#-handler-options)
-  - [... dynamic diagnostics](#-dynamic-diagnostics)
-  - [... syntax highlighting](#-syntax-highlighting)
-  - [... collection of labels](#-collection-of-labels)
-- [Acknowledgements](#acknowledgements)
-- [License](#license)
+- [`miette`](#miette)
```

Comment:
is this nesting because you auto-generated the ToC? Is it possible to unnest this?